wiki/Tournaments/TWC/2018/en.md

```diff
@@ -0,0 +1,239 @@
+  - If the amount of registered countries is below 32, the number might be reduced to 24, 20 or 16. The aim is always to let as many countries participate as possible!
+
+5. Mapset selectors may not participate as a player in this tournament.
+
+### Stage Instructions
+
+1. In the first stage (Group Stage), the teams will be divided into 8 groups of 3 teams.
+  - This may change depending on how many teams are accepted into the competition at the end.
+
+2. All the teams from each group will face each other.
+3. Rankings of each group are determined by sorting the results of each team's performance in the following priority:
+   - Most matches won.
+   - Have higher `{(the number of beatmaps won) - (the number of beatmaps defeated)}`.
+   - Most beatmaps won.
+   - **Winner of the match played previously between the tied teams.**
+   - In the event of a triple tie:
+     - Have higher `∑{(total score difference) / (maximum score)}`.
+     - Winner of the rematch.
+
+4. The top 2 teams of each group will move on to the Knock-Out Stages.
+  -  This may change with the actual Group Stage setup.
+
+5. Following stages are Double Elimination Stages. This means that the winner moves to the next stage and the losing team gets moved to the Loser bracket.
+6. Based on [this image](https://puu.sh/bUq5V/f1066103b0.png), the stages are split up into the following:
+
+| Stage | Match ID |
+| -----: | -------- |
+| Round of 16 |  A, B, C, D, E, F, G, H |
+| Quarterfinals | I, J, K, L & R, S, T, U |
+| Semifinals | M, N & V, W, X, Y, Z, AA |
+| Finals | O & AB, AC, AD |
+| Grand Finals | AE, P, Q |
+
+**Winning conditions:**
+   - In Group Stage, you need to win 4 beatmaps to win a match. (Best-of-7)
+   - In the Round of 16 and the Quarterfinals, you need to win 5 beatmaps to win a match. (Best-of-9)
+   - In Semifinals, you need to win 6 beatmaps to win a match. (Best-of-11)
+   - In the **Finals and Grand Finals**, you need to win 7 beatmaps to win a match. (Best-of-13)
+
+### Match Instructions
+
+1. A referee will create a multiplayer room 15 minutes in advance. Players must gather during this period.
+  - Room settings are osu!taiko, Team-Vs., Win Condition: 'Score'. Room name must be "TWC 2017: (TeamBlue) vs (TeamRed)".
+  - The team mentioned first in the room name must be the blue team, the team mentioned second in the room name must be the red team.
+
+2. Players are free to select up to two warm-up maps. Using beatmaps with questionable content is prohibited. All maps must be osu!taiko specific maps.
+3. Each captain can ban **one beatmap** to be selected from the pool. These beatmaps are not allowed to be picked by any team in the entire match.  
+4. Beatmap selection will alternate between each captain selecting a beatmap out of the map pool.
+5. Each captain must use "!roll" once in #multiplayer.
+  - The winner of the !roll starts picking the first beatmap of the match.
+  - The loser of the !roll starts banning.
```

Comment:
...followed by the winner of the !roll to ban a beatmap.

Reply:
dont think this is needed to be honest.